apps/backoffice-v2/src/domains/alerts/query-keys.ts

```diff
@@ -35,4 +39,8 @@ export const alertsQueryKeys = createQueryKeys('alerts', {
       queryFn: () => fetchAlertDefinitionByAlertId({ alertId }),
     };
   },
+  alertLabels: () => ({
+    queryKey: ['alertLabels'],
```

Comment:
Pretty sure the key of the method in this case `alertLabels` is already applied by the library.
```suggestion
    queryKey: [{}],
```